code/modules/research/rdconsole.dm

```diff
@@ -460,6 +460,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 	if(machine)
 		if(enough_materials && being_built)
 			for(var/i in 1 to amount)
+				SSblackbox.record_feedback("nested tally", "RND Production List", 1, list("[being_built.category]", "[being_built.name]"))
```

Comment:
I think this might work better
```suggestion
			SSblackbox.record_feedback("nested tally", "RND Production List", amount, list("[being_built.category]", "[being_built.name]"))
			for(var/i in 1 to amount)
```

Reply:
good point